typescript/ts-converter/src/ast/AstExpressionConverter.ts

```diff
@@ -324,6 +346,42 @@ export class AstExpressionConverter {
         }
     }
 
+    convertTemplateExpression(expression: ts.TemplateExpression): Expression {
+        let tokens: Array<TemplateTokenDeclaration> = [];
+
+        let head = expression.head.getText();
+        // head = "`headTrimmed${"
```

Comment:
I you sure there's no better API?

Reply:
There is! Fixed.

Reply:
Side note: solution here is to change `.getText()` to just `.text`, the later returns parsed version of literal, while `.getText()` returns literal as is. The thing is, almost everywhere in this file we use `.getText()` to retrieve literals, which means, for example, that string literals are returned with quote marks (`"aaa"` instead of more logical `aaa`). I think, we should get rid of `getText()` in our parser at all. Let's discuss.